src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimIssuedStaffNotificationService.java

```diff
@@ -28,91 +32,119 @@
+    private EmailData prepareEmailData(final Claim claim,
+                                       final Optional<String> defendantPin,
+                                       final String submitterEmail,
+                                       final String authorisation) {
         EmailContent emailContent = provider.createContent(claim);
         return new EmailData(staffEmailProperties.getRecipient(),
             emailContent.getSubject(),
             emailContent.getBody(),
-            getAttachments(claim, defendantPin, submitterEmail));
+            getAttachments(claim, defendantPin, submitterEmail, authorisation));
     }
 
     private List<EmailAttachment> getAttachments(
         final Claim claim,
         final Optional<String> defendantPin,
-        final String submitterEmail
+        final String submitterEmail,
+        final String authorisation
     ) {
         final List<EmailAttachment> emailAttachments = new ArrayList<>();
 
         if (!claim.getClaimData().isClaimantRepresented()) {
             String pin = defendantPin.orElseThrow(NullPointerException::new);
-            emailAttachments.add(sealedClaimPdf(claim, submitterEmail));
+            emailAttachments.add(sealedClaimPdf(authorisation, claim, submitterEmail));
             emailAttachments.add(defendantPinLetterPdf(claim, pin));
         } else {
-            emailAttachments.add(sealedLegalClaimPdf(claim));
+            emailAttachments.add(sealedLegalClaimPdf(authorisation, claim));
         }
 
         return emailAttachments;
     }
 
-    private EmailAttachment sealedLegalClaimPdf(Claim claim) {
-        byte[] generatedPdf = pdfServiceClient.generateFromHtml(
-            staffEmailProperties.getEmailTemplates().getLegalSealedClaim(),
-            legalSealedClaimContentProvider.createContent(claim)
-        );
+    private EmailAttachment sealedLegalClaimPdf(final String authorisation, final Claim claim) {
+        byte[] generatedPdf = getSealedClaimPdfDocument(authorisation, claim, null);
 
         return pdf(
             generatedPdf,
             format("%s-sealed-claim.pdf", claim.getReferenceNumber())
         );
     }
 
-    private EmailAttachment sealedClaimPdf(Claim claim, String submitterEmail) {
-        byte[] generatedPdf = pdfServiceClient.generateFromHtml(
-            staffEmailProperties.getEmailTemplates().getSealedClaim(),
-            sealedClaimContentProvider.createContent(claim, submitterEmail)
-        );
+    private byte[] getSealedClaimPdfDocument(final String authorisation, final Claim claim,
+                                             final String submitterEmail) {
+        if (documentManagementFeatureEnabled && claim.getSealedClaimDocumentManagementSelfPath().isPresent()) {
+            return documentManagementService.downloadDocument(authorisation,
+                claim.getSealedClaimDocumentManagementSelfPath().get());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3655)